### ISSUE TITLE
PAYMENT CLEARANCE THREAD

### ISSUE THREAD

**Issue by Morningstar88** [Morningstar88/X#7]
Please do not start a job until you get payment approved here, on Github issues. 

You may use this 2 Question Form:

## What Is The Price?

After you get an approval, in writing, on this thread only... you can go and order the job.

Please don't arrange jobs on emails, as it can be confusing.

Thank you for your kind cooperation.

Mgt.

**Comment by sadrev25** [comment]
Actually,I hired a js developer for that issue that was not opening in mobile right.Hired person fixed that problem,After a payment we will get a solved code.45 usd for that issue solver.

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Thanks for coming to Github...
Sorry, I don't remember approving that payment.

Do you have an email where I approved it?

Who is supposed to pay?

**Comment by sadrev25** [comment]
Yeah okay hereafter after the payment clearence in github, I will proceed.sorry for inconvenience but this time I assigned a work to js developer already,is fixed that problem within 6hours .so we need to consider his payment.

**Comment by sadrev25** [comment]
Then jake,we need to think about business model to move further.

**Comment by Morningstar88** [comment]
Multiple emails made the conversation difficult and unclear. 

I don't have any money left as I am moving.

I will find 45USD, then pause cooperation until I have more money. I could have got that issue fixed for 10-20USD on Fiverr. 

I thought the payment was for a Marketer. Let me check, as you might be right.

**Comment by Morningstar88** [comment]
OK, I checked the emails.

I did say yes... but I thought it was about Marketing.

This is why we talk on Github, so we are clear what issue it is.

Give me two days to find the money, please forward his Paypal or payment details.

**Comment by sadrev25** [comment]
Yeah.That guy agreed working for us.hereafter any work we will approach him in a free of cost. You only asked about js frontend developer ,for future this deal be best!

**Comment by sadrev25** [comment]
Ok no problem jake ,now will give 20 usd .After that if we have any problem will approach him.is it ok ?

**Comment by sadrev25** [comment]
> OK, I checked the emails. I did say yes... but I thought it was about Marketing. This is why we talk on Github, so we are clear what issue it is. Give me two days to find the money, please forward his Paypal or payment details.

Okay jake

**Comment by sadrev25** [comment]
Here after we will talk in guthub jake

**Comment by Morningstar88** [comment]
20 USD is fine... thanks.

And it's not Guthub, it's Github... haha. Gut means big belly. 

In future we will ask open source volunteers first. 

It's quicker and cheaper. 

Anyway, well done for solving the issue so quick.

**Comment by sadrev25** [comment]
In paypal money conversion rate is high jake to inr.After you arranged money ,put a message here.I will send him money in inr and get ntd from you coming Saturday.Is it ok

**Comment by Morningstar88** [comment]
You want me to pay you in TWD?

Sure, that's fine. 
Now lets brainstorm the business model.

**Comment by Morningstar88** [comment]
Business model thread here:

https://github.com/Morningstar88/X/issues/6

We will stick on that as a priority issue.

**Comment by sadrev25** [comment]
Yeah fine.Today night I will send business model !

**Comment by sadrev25** [comment]
> Business model thread here: #6 We will stick on that as a priority issue.

Offcourse
